src/nvim/normal.c

```diff
@@ -2598,10 +2598,8 @@ do_mouse (
         end_visual.col = leftcol;
       else
         end_visual.col = rightcol;
-      if (curwin->w_cursor.lnum <
+      if (curwin->w_cursor.lnum >=
```

Comment:
I can't wrap my head around this, probably not from coverity, and I can only assume Bram knew what he was doing.